script.ts

```diff
@@ -1,99 +1,108 @@
+const prisma = new PrismaClient();
 
 async function main() {
-
-  await prisma.job_table.deleteMany() // delete the existing data everytime
-  await prisma.department_table.deleteMany() // delete the existing data everytime
-  await prisma.location_table.deleteMany() // delete the existing data everytime
-  await prisma.hr_employees_table.deleteMany() // delete the existing data everytime
+  // await prisma.job_table.deleteMany() // delete the existing data everytime
+  // await prisma.department_table.deleteMany() // delete the existing data everytime
+  // await prisma.location_table.deleteMany() // delete the existing data everytime
+  // await prisma.hr_employees_table.deleteMany() // delete the existing data everytime
   const manager = await prisma.hr_employees_table.create({
     data: {
       first_name: 'John',
       last_name: 'Doe',
       email: 'john.doe@example.com',
       phone_number: '1234567890',
-      job_id: 1, 
+      job_id: 1,
       department_id: 1,
-      location_id: 1, 
+      location_id: 1,
       starting_date: new Date('2023-01-01'),
-      role: 'ADMIN', 
+      role: 'ADMIN',
     },
-  })
+  });
 
   const users = await prisma.hr_employees_table.createMany({
     data: [
       {
-        first_name: "Alice",
-        last_name: "Johnson",
-        email: "alice.johnson@example.com",
-        phone_number: "123451231",
-        job_id: 1, 
+        first_name: 'Alice',
+        last_name: 'Johnson',
+        email: 'alice.johnson@example.com',
+        phone_number: '123451231',
+        job_id: 1,
         manager_id: null,
         department_id: 1,
         location_id: 1,
-        starting_date: new Date("2023-01-01"),
+        starting_date: new Date('2023-01-01'),
         role: 'BASIC',
       },
       {
-        first_name: "Bob",
-        last_name: "Smith",
-        email: "bob.smith@example.com",
-        phone_number: "123123464",
+        first_name: 'Bob',
+        last_name: 'Smith',
+        email: 'bob.smith@example.com',
+        phone_number: '123123464',
         job_id: 2,
         manager_id: null, // TODO: this is not working
         department_id: 2,
         location_id: 2,
-        starting_date: new Date("2023-02-01"),
+        starting_date: new Date('2023-02-01'),
         role: 'ADMIN',
       },
-
     ],
-})
+  });
 
-    const departments = await prisma.department_table.createMany({
-      data: [
-        { department_name: "Engineering" },
-        { department_name: "Human Resources" },
-        { department_name: "Marketing" },
-        { department_name: "Sales" },
-        { department_name: "Customer Support" },
-      ],
-    })
+  // const departments = await prisma.department_table.createMany({
+  //   data: [
+  //     { department_name: 'Engineering' },
+  //     { department_name: 'Human Resources' },
+  //     { department_name: 'Marketing' },
+  //     { department_name: 'Sales' },
+  //     { department_name: 'Customer Support' },
+  //   ],
+  // });
```

Comment:
Careful with leaving in the commented out code.

Reply:
We have a reason for this. We can loop you in on site and fix it later.